wireapp/wire-android
# fix: disable back press when app lock team enforced (WPB-5644)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-5644" title="WPB-5644" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />WPB-5644</a>  [Android] Back button bypasses applock screen when it is forced
  </td></table>
  <br />
 














----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

When AppLock is enforced by team, it was able to press back button (either hardware or app button) and navigate back in the screen and use the app normally until next closing/opening app and being prompted with setting app lock password.

### Solutions

When AppLock is enforced by team, hide back button on app and disable back button on screen.
Functionalities remain the same when it is not enforced by team but user wants to set AppLock.

### Testing

#### How to Test

- Open App
- Receive AppLock enforced by team dialog
- on Set AppLock password screen
- make sure back button is hidden (top left) + hardware back button has no effect.
